pkg/proxy/apis/config/scheme/testdata/KubeProxyConfiguration/after/v1alpha2.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: kubeproxy.config.k8s.io/v1alpha2
+bindAddress: 0.0.0.0
+bindAddressHardFail: false
+clientConnection:
+  acceptContentTypes: ""
+  burst: 10
+  contentType: application/vnd.kubernetes.protobuf
+  kubeconfig: ""
+  qps: 5
+configSyncPeriod: 15m0s
+detectLocal:
+  bridgeInterface: ""
+  clusterCIDR: ""
+  interfaceNamePrefix: ""
+detectLocalMode: ""
+enableProfiling: false
+healthzBindAddress: 0.0.0.0:10256
+hostnameOverride: ""
+iptables:
+  localhostNodePorts: true
```

Comment:
So one of the ideas was that we were going to change the defaults in v1alpha2, eg to make `localhostNodePorts` `false`. (But this is one of the things that would be discussed in the KEP update that, as @neolit123 mentioned, we still need to do.)